daemon/prune.go

```diff
@@ -234,3 +269,23 @@ func (daemon *Daemon) NetworksPrune(pruneFilters filters.Args) (*types.NetworksP
 	}
 	return rep, err
 }
+
+func getUntilFromPruneFilters(pruneFilters filters.Args) (time.Time, error) {
+	until := time.Time{}
+	if pruneFilters.Include("until") {
+		untilFilters := pruneFilters.Get("until")
+		if len(untilFilters) > 1 {
+			return until, fmt.Errorf("more than one until filter specified")
+		}
+		ts, err := timetypes.GetTimestamp(untilFilters[0], time.Now())
+		if err != nil {
+			return until, err
+		}
```

Comment:
This seems to be inconsistent with `events`. The timestamp is parsed on the client in `client/events.go`.  Should they be consistent?

The API should probably be more strict about what it accepts in this filter.